ote/src/cljs/ote/views/front_page.cljs

```diff
@@ -11,46 +11,80 @@
             [ote.db.common :as common]
             [ote.localization :refer [tr tr-key]]
             [ote.db.transport-service :as t-service]
-            [ote.db.transport-operator :as t-operator]))
+            [ote.db.transport-operator :as t-operator]
+            [cljs-time.core :as time]
+            [cljs-time.format :as format]
+            [cljs-time.coerce :as coerce]
+            ))
 
-(defn transport-services-listing [e! transport-operator-id services]
-  (when services
+(def custom-formatter (format/formatter "dd.MM.yy H:i:s"))
+
+(defn format-my-time [my-time]
```

Comment:
what is "my time"? do you mean local time? Also, this should be `ote.time` namespace